python/cudf/cudf/core/reshape.py

```diff
@@ -58,6 +58,7 @@ def concat(objs, axis=0, ignore_index=False, sort=None):
     else:
         axis = param_axis
 
+    ignore_duplicate_columns = kwargs.get("ignore_duplicate_columns", False)
```

Comment:
Any reason not to just always check it? We don't support duplicated column names period.

Reply:
Had a discussion offline with @kkraus14 and fixed quadrant construction logic in `get_renderable_dataframe`.  The actual issue was that the existing logic is actually creating `upper_left` and `upper_right` with intersecting columns, which will cause the `concat` to fail. Hence fixed the quadrant logic to have non-intersecting columns in `upper_left` - `upper_right` and `lower_left` - `lower_right`.